### DISCUSSION TITLE
DNS Configuration

### DISCUSSION THREAD

**Discussion by KhalilFaizi** [Diamond-Family/DNS-Installition-and-Configution#6] [Q&A] [answered]
**We installed DNS server.
What do you think about DNS configuration?**

**Comment by mokhtar003** [comment, marked as answer]
هو ښه نظر دی دا که چیري په پروژه کي عملی سي

**Reply by KhalilFaizi** [reply]
Ok

**Reply by mokhtar003** [reply]
Tanx dear

**Reply by Abdulmateenmaher** [reply]
well

**Reply by Abdulmateenmaher** [reply]
yes

**Reply by Abdulmateenmaher** [reply]
good

**Comment by Farhadzahin123** [comment]
هو ښه نظر دي که چیر یی په پروژه کي عملي شی

**Reply by KhalilFaizi** [reply]
Ok

**Comment by Farhadzahin123** [comment]
سلام

**Reply by KhalilFaizi** [reply]
Walikum salam 
What is your idea about this issue?

**Reply by mokhtar003** [reply]
علیک سلام تا سنگه کرل خپل کارونه خلاص سول

**Reply by KhalilFaizi** [reply]
Ok

**Reply by mokhtar003** [reply]
I think it's  better to first we install and after it configuration

**Reply by Farhadzahin123** [reply]
په issue کولا یشو چي د خپلی پروژی معلومات ذخیره کړو

**Comment by Abdulmateenmaher** [comment]
great idea!
configuration is our main issue.
if possible so do.

**Reply by Abdulmateenmaher** [reply]
good

**Comment by Abdulmateenmaher** [comment]
we will do that in next version
